tests/microos/disk_boot.pm

```diff
@@ -29,7 +29,7 @@ sub run {
         shift->wait_boot(bootloader_time => 300);
     }
     microos_login;
-    record_kernel_audit_messages(log_upload => 1);
+    record_kernel_audit_messages(log_upload => 1) unless (get_var('NICTYPE') eq 'tap');
```

Comment:
Why are we skipping this step? Is it failing?  
If so, please add a comment so that our future selves will remember :slightly_smiling_face: